charts/templates/ingress.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Values.ingress.name }}
+  annotations:
```

Comment:
Maybe we could move annotations to values.yaml? not every everyone is using nginx